### PR TITLE
Set peer ID to start with "-WD-"

### PR DESCRIPTION
**Pull request by feross**
To distinguish WebTorrent Desktop (WD) from WebTorrent in the browser
(WW).

See the spec:

http://www.bittorrent.org/beps/bep_0020.html
https://wiki.theory.org/BitTorrentSpecification

### CODE REVIEW

File: src/renderer/webtorrent.js [hunk 1]
```diff
@@ -26,9 +28,36 @@ global.WEBTORRENT_ANNOUNCE = defaultAnnounceList
...
+
+/**
+ * Version number in Azureus-style. Generated from major and minor semver version.
+ * For example:
+ *   '0.16.1' -> '0016'
+ *   '1.2.5' -> '0102'
+ */
+var VERSION_STR = VERSION.match(/([0-9]+)/g)
+  .slice(0, 2)
+  .map(function (v) { return zeroFill(2, v) })
```
Comment (mathiasvr) [inline src/renderer/webtorrent.js]: Nitpick: Use arrow function: `.map((v) => zeroFill(2, v))`.

**Comment by mathiasvr** [comment]
LGTM

File: src/renderer/webtorrent.js (hunk 1 above)
Comment (feross) [inline src/renderer/webtorrent.js]: Will do!

**Comment by yciabaud** [comment]
Nice!

**Pull request merged**